bin/autosign-validator

```diff
@@ -2,6 +2,8 @@
 require 'autosign'
 require 'logging'
 
+raw_csr = $stdin.read
```

Comment:
Could you add a comment (the old one is  fine) explaining why it needs to read all the input?